docs/import-from-github.mdx

```diff
@@ -1,54 +1,80 @@
 ---
-title: Import from GitHub
-description: 'Import your machines from GitHub into Stately Studio.'
+title: Connect GitHub repo
+description: 'Sync your machines from GitHub with Stately Studio.'
 ---
 
-There are currently two ways to import your machines from GitHub into Stately Studio:
-
-1. [Import all machines from a GitHub repository](#import-all-machines-from-a-github-repository).
-2. [Import machines from a GitHub URL.](#import-machine-from-a-github-url)
+You can connect a GitHub repo to a new project in Stately Studio, keeping updates between GitHub and Stately Studio in sync. Connecting a GitHub repo allows you to import your existing machines from GitHub and push changes to your machines back to your repo as pull requests.
 
 :::studio
 
-Import from GitHub is a [premium feature](/studio-pro-plan.mdx) of Stately Studio.
-
-We offer a **free trial** on the [Stately Studio Pro account](https://stately.ai/pricing) so you can explore how our Pro features work for you and your team.
+Connect GitHub repo is a premium feature of Stately Studio. You can try Stately Studio’s premium plans with a free trial. [Check out the features on our Pro plan](studio-pro-plan.mdx), [Team plan](studio-team-plan.mdx), [Enterprise plan](studio-enterprise-plan.mdx) or [upgrade your existing plan](https://stately.ai/registry/billing).
 
 :::
 
-## Import all machines from a GitHub repository
+## Connect GitHub repo
 
-Use **Import from GitHub** to import all the machines from your GitHub repository into one project in the Stately Studio.
+There are three steps to setting up your GitHub repo as a connected project:
 
-:::info
+1. Select the repo. You can choose the repos based on your GitHub permissions.
```

Comment:
I love the clarity of just 3 simple steps to get started here and notably they align with the "Steps 1/3" in the modal UI for the feature. However, I didn't see where we actually mention "Connect GitHub repo" as a button or where to find that button. Do we want to start with something like this to kick it off?

"Upon clicking the Connect GitHub repo button, just above your projects on the My Projects or Team Projects pages, there are three steps to setting up your GitHub repo as a connected project."

If that's too long, we can cut out some words.

"On the My Projects or Team Projects pages, click the Connect GitHub repo button and follow these three steps to set up your GitHub repo as a connected project."